package.json

```diff
@@ -13,7 +13,13 @@
     "db:migrate:deploy": "nx run-many --target=db:migrate:deploy",
     "prepare": "husky install",
     "precommit": "lint-staged --relative",
-    "help": "nx help"
+    "help": "nx help",
+    "serve:server": "npx nx serve amplication-server",
+    "serve:client": "npx nx serve amplication-client",
+    "serve:dsg": "nx run-many --parallel --target=serve --projects=amplication-build-manager,local-data-service-generator-controller",
+    "serve:git": "nx run-many --parallel --target=serve --projects=amplication-git-pull-request-service,local-data-service-generator-controller",
+    "serve:storage": "npx nx serve amplication-storage-gateway",
+    "serve:plugins": "npx nx serve amplication-plugin-api"
```

Comment:
I get the point of facilitating the user understanding but it is not a good practice mix packages scripts; since we are using nx monorepo we should use root commands in nx.json.